.github/workflows/rust.yml

```diff
@@ -112,13 +77,23 @@ jobs:
       matrix:
         include:
         - os: macos-latest
+          # macos-latest seems to already run on arm64(=aarch64):
+          # https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job#standard-github-hosted-runners-for-public-repositories
           TARGET: aarch64-apple-darwin
 
+        - os: macos-latest
```

Comment:
Are you able to run such binaries ? macos-latest appears to be broken at the moment and macos-13 was the only solution at the moment.

Reply:
How was it broken? The thing is, in the original state everything was run through cross. This could very well break things.

You could perhaps test the binary from my app (it doesnt do a lot, but hopefully runs): https://github.com/9SMTM6/mcmc-demo/releases/tag/main

Reply:
According to my tests it runs.